### DISCUSSION TITLE
Vector image not displaying correctly

### DISCUSSION THREAD

**Discussion by Advocas** [arkypita/LaserGRBL#2077] [Q&A]
Please help with this, I created the following image in Inkscape, when I open it in LaserGRBL it doesn't display properly see images below. I have done this many times before without issue,

![image](https://github.com/arkypita/LaserGRBL/assets/89773859/486d5576-3d84-4b5f-9664-7155a85ee29f)

![image](https://github.com/arkypita/LaserGRBL/assets/89773859/0d5d7ce0-b761-456e-96d1-7a6867a8d93e)

I'm running Inkscape 1.2.2 and LaserGRBL v4.9.4

**Comment by Advocas** [comment]
I tested the image in an online Vector editor (https://pixelied.com), the image showed correctly.
![image](https://github.com/arkypita/LaserGRBL/assets/89773859/11bd532b-b566-4c02-a46e-95e9d257431f)
I also tried it in Lightburn and again it showed correctly
![image](https://github.com/arkypita/LaserGRBL/assets/89773859/943cafc0-599d-448b-a1fb-6eed6afb1ade)
Could this be a bug in LaserGRBL?
I could engrave it using Lightburn but I prefer LaserGRBL and only use Lightburn when I absolutly have to.

**Comment by Tribo-2dot0** [comment]
Pode postar o .svg para eu fazer uma verificação?

**Reply by Advocas** [reply]
There you go,
![Test](https://github.com/arkypita/LaserGRBL/assets/89773859/2ef830a7-3ba1-4a75-8249-3c54f53fc586)

**Comment by Advocas** [comment]
I found a way to fix it, if I "Break-Apart" then save, it displays correctly but when I "Combine" it again it reverts back to the broken state. I always "Combine" when I have finished an image, I don't understand why this one is different.
